docs/src/main/tut/freemonad.md

```diff
@@ -166,14 +166,14 @@ DSL. By itself, this DSL only represents a sequence of operations
 To do this, we will use a *natural transformation* between type
 containers.  Natural transformations go between types like `F[_]` and
 `G[_]` (this particular transformation would be written as
-`NaturalTransformation[F,G]` or as done here using the symbolic
+`FunctionK[F,G]` or as done here using the symbolic
 alternative as `F ~> G`).
```

Comment:
I would say that we can describe the type `FunctionK` as a *natural transformation*. So, when we would put the term in backticks (i.e. code) we'd say `FunctionK` but otherwise we'd say transformation or natural transformation. Does this make sense?